src/Data/List/Base.agda

```diff
@@ -369,45 +369,37 @@ removeAt (x ∷ xs) (suc i)  = x ∷ removeAt xs i
 
 takeWhile : ∀ {P : Pred A p} → Decidable P → List A → List A
 takeWhile P? []       = []
-takeWhile P? (x ∷ xs) with does (P? x)
-... | true  = x ∷ takeWhile P? xs
-... | false = []
+takeWhile P? (x ∷ xs) = if does (P? x) then x ∷ takeWhile P? xs else []
```

Comment:
In general, can we put the `then` and `else` on separate lines?

Reply:
Will do once I'm no longer on the move...

Reply:
Done.